lib/trento/infrastructure/prometheus/adapter/mock_prometheus_api.ex

```diff
@@ -3,7 +3,36 @@ defmodule Trento.Infrastructure.Prometheus.MockPrometheusApi do
   Mocks prometheus API calls
   """
 
+  alias Trento.Charts.ChartTimeSeriesSample
+
   @behaviour Trento.Infrastructure.Prometheus.Gen
+  @behaviour Trento.Charts.HostDataFetcher
+
+  def cpu_busy_iowait(_, from, to), do: random_chart_data(from, to)
+  def cpu_idle(_, from, to), do: random_chart_data(from, to)
+  def cpu_busy_system(_, from, to), do: random_chart_data(from, to)
+  def cpu_busy_user(_, from, to), do: random_chart_data(from, to)
+  def cpu_busy_other(_, from, to), do: random_chart_data(from, to)
+  def cpu_busy_irqs(_, from, to), do: random_chart_data(from, to)
+  def ram_total(_, from, to), do: random_chart_data(from, to, 0..30_000)
+  def ram_used(_, from, to), do: random_chart_data(from, to, 0..30_000)
+  def ram_cache_and_buffer(_, from, to), do: random_chart_data(from, to, 0..30_000)
+  def ram_free(_, from, to), do: random_chart_data(from, to, 0..30_000)
+  def swap_used(_, from, to), do: random_chart_data(from, to, 0..30_000)
+  def num_cpus(_, _), do: {:ok, 8}
```

Comment:
Does this fixed value work?
Shouldn't the samples number go increasing?

Reply:
The samples number are always the same, one each 5 minute, in the real api we use one point each 60 seconds or something like that